cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -128,6 +129,12 @@ func createStaticPodFiles(manifestDir string, cfg *kubeadmapi.MasterConfiguratio
 		return err
 	}
 
+	// create audit log file for apiserver
+	if err := audit.CreateDefaultAuditLogPolicy(cfg.AuditPolicyDir); err != nil {
```

Comment:
People are going to want to knob paths here for policy on logging.  Given it's beta I'm debating whether we should feature gate it or just provide a 1st class override. 

/cc @roberthbailey @liggitt .

Reply:
seems like a natural point to drop in a custom policy, especially since nothing else in kubeadm is really functionally dependent on this particular default audit policy